site/en/docs/webstore/cws-dashboard-distribution/index.md

```diff
@@ -90,13 +90,35 @@ the Chrome Web Store. You can either:
 When you publish to the world, your item will be immediately visible to the region(s) you select,
 and it'll be visible in the store's search results.
 
+## Managing a test version and a production version
```

Comment:
This section looks like a good clarification on how to publish test versions. As discussed this morning, I think it's best for us to tackle white-labeling in a separate, more targeted CUJ.

Reply:
Sounds good!